src/Benchmarks/Data/RawDb.cs

```diff
@@ -88,6 +88,45 @@ public RawDb(IRandom random, DbProviderFactory dbProviderFactory, IOptions<AppSe
             return result;
         }
 
+        public async Task<World[]> LoadMultipleUpdatesRows(int count)
+        {
+            var result = await LoadMultipleQueriesRows(count);
+
+            using (var db = _dbProviderFactory.CreateConnection())
```

Comment:
I think we should use a single DB connection since the spec doesn't say otherwise, and it may be faster.